include/swift/AST/DiagnosticsSema.def

```diff
@@ -562,6 +562,9 @@ ERROR(cannot_return_value_from_void_func,none,
 
 ERROR(sema_no_import,Fatal,
       "no such module '%0'", (StringRef))
+NOTE(sema_no_import_arch,Fatal,
+      "could not find module '%0' for architecture '%1' "
+      "found: %2", (StringRef, StringRef, StringRef))
```

Comment:
I like this touch of including the architectures that *are* present! Good thinking.

Reply:
@brentdax's suggestion 😄